sources/Chizkuny/Chizkuni_Devarim_parse.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+import sys
+import os
+# for a script located two directories below this file
+p = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+sys.path.insert(0, p)
+sys.path.append("/Users/Eytan/Desktop/sefaria/Sefaria-Data/sources")
```

Comment:
There are two big problems with this line. 

1. It uses an absolute path, instead of a relative one. The structure of Sefaria-Data is constant across systems, so make references relative to the root file of Sefaria Data.

2.  The sources folder is already accessible from the Sefaria-Data root. Adding it to the python path is messy and a potential source for errors, especially if this script needs to be run on a separate machine.